src/buffers/RecordingBuffer.cpp

```diff
@@ -21,14 +21,16 @@
 
 #include "RecordingBuffer.h"
 
+#include "libXBMC_pvr.h"
```

Comment:
This shouldn't be needed.  RecordingBuffer.h includes Buffer.h, which includes client.h, which includes this file.

Reply:
Ok removed